### ISSUE TITLE
In manage column popup, the order change buttons should be readable by Jaws screen reader

### ISSUE THREAD

**Issue by irfanlbrce** [asitparida/VxGrid#106]
In manage column popup, the order change buttons should be readable by Jaws screen reader

**Comment by asitparida** [comment]
this is done
